### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by KevinEtienne**

### CODE REVIEW

**Comment by KevinEtienne** [comment]
@adam-thomas can you review/merge please?

File: setup.cfg [hunk 1]
```diff
@@ -5,4 +5,4 @@ max-line-length = 90
 exclude = *migrations*
 statistics = true
 application-import-names = user_management
-import-order-style = google
+import-order-style = smarkets
```
Comment (adam-thomas) [inline setup.cfg]: what's this? :)

**Comment by adam-thomas** [comment]
LGTM

File: setup.cfg (hunk 1 above)
Comment (KevinEtienne) [inline setup.cfg]: It has been added in https://github.com/public/flake8-import-order/pull/57

> smarkets - style as google only with import statements before from X import

Comment (adam-thomas) [inline setup.cfg]: oh cool :)

**Pull request merged**